README.md

```diff
@@ -61,9 +61,10 @@ Here is the list of available major versions of the library (as of 2024-08-16):
 | v5.1    | 5.0       |                   |            | Unmaintained      |
 | v6.0    | 6.0       |                   |            | Unmaintained      |
 | v6.1    | 6.0       |                   |            | Unmaintained      |
-| v7.0    | 7.0       |                   | &#x2714;   |                   |
-| v8.0    | 8.0       |                   | &#x2714;   |                   |
-| v8.1    | 8.0       | &#x2714;          | &#x2714;   |                   |
+| v7.0    | 7.0       |                   |            | Unmaintained      |
+| v8.0    | 8.0       |                   |            | Unmaintained      |
+| v8.1    | 8.0       |                   | &#x2714;   |                   |
```

Comment:
Aren't you about to merge .NET 9.0 into v8.1 branch?

Reply:
NVM I guess

Reply:
Yeah, almost made the mistake. Now it is fixed ^^'